ISISScientificComputing/autoreduce
# Help page index menu

### Summary of work
Created a side menu/index for topics displayed on Autoreduction Webapp Help page.

![image](https://user-images.githubusercontent.com/5902731/89915294-1472f900-dbee-11ea-90d3-d6417cfefaee.png)

And on a small screen size:

![image](https://user-images.githubusercontent.com/5902731/89915379-30769a80-dbee-11ea-874b-2e41a79ef4d1.png)

### How to test your work
* Code review
* Index/side mentu is well placed and structured
* All links navigate user to correct section of help page.
* Menu links work with filters
* Ensure the page functions and looks good on different screen sizes

### Additional comments
* I wasn't sure how to make anchor links work on page load since the anchor links are generated by some JS code which is run after the page would scroll to any anchor link.

Fixes #677


**Before merging ensure the release notes have been updated**
